common.go

```diff
@@ -0,0 +1,72 @@
+package vmcommon
+
+const tickerMinLength = 3
```

Comment:
maybe move the `metaESDT` type here as well? 
see this todo: https://github.com/ElrondNetwork/elrond-go/blob/4c334e98e55c33625b4c6fead7c4bf28de3f6692/vm/systemSmartContracts/esdt.go#L556

also, there are duplicated declarations between the two files (tickerMaxLength for example)

Reply:
need another cleanup where we move a set of things to core.